src/check-comments.js

```diff
@@ -0,0 +1,14 @@
+function checkComments (context) {
```

Comment:
This name doesn't describe what this function do.

Also it is not obvious why it is in `src`